build.gradle.kts

```diff
@@ -15,5 +16,21 @@ buildscript {
 }
 
 allprojects {
+    apply(plugin = rootProject.libs.plugins.kover.get().pluginId).also {
+        koverMerged {
+            filters {
+                annotations {
+                    excludes += listOf(
+                        "androidx.compose.ui.tooling.preview.Preview",
+                        "androidx.compose.runtime.Composable"
+                    )
```

Comment:
❓ Why these annotations are excluded?

Reply:
Because unit tests will not be targeting to the Composables or the Previews, and we don't want them to be considered when the overall unit test coverage is being calculated 🙂

Reply:
Makes sense, thanks! 🙂